pkg/apis/config/default.go

```diff
@@ -191,6 +197,14 @@ func NewDefaultsFromMap(cfgMap map[string]string) (*Defaults, error) {
 		tc.DefaultContainerResourceRequirements = resourceRequirementsValue
 	}
 
+	if defaultImagePullBackOff, ok := cfgMap[defaultImagePullBackOffTimeout]; ok {
```

Comment:
If we go with a time representation (see comment above), we can use [`time.Duration`](https://pkg.go.dev/time#Duration) for the type and [`time.ParseDuration`](https://pkg.go.dev/time#ParseDuration) to parse it.